models/wrf/model_mod.f90

```diff
@@ -7402,7 +7402,7 @@ subroutine fill_default_state_table(default_table)
                           'UPDATE                    ', &
                           '999                       '  /)
 row = row+1
-default_table(:,row) = (/ 'T                         ', &
+default_table(:,row) = (/ 'THM                       ', &
```

Comment:
note, T vs THM is a wrf v3 to v4 issue. T is prognostic in v4.

Inside WRF, use_theta_m=0 for T= THM